static/app/views/organizationIntegrations/sentryAppExternalForm.tsx

```diff
@@ -15,8 +15,9 @@ import withApi from 'sentry/utils/withApi';
 // 0 is a valid choice but empty string, undefined, and null are not
 const hasValue = value => !!value || value === 0;
 
+// See docs: https://docs.sentry.io/product/integrations/integration-platform/ui-components/formfield/
 export type FieldFromSchema = Omit<Field, 'choices' | 'type'> & {
-  type: 'select' | 'textarea' | 'text';
+  type: string;
```

Comment:
Is this because even if it's a select field you're ultimately selecting a string?